yak/cmake/yak-config.cmake.in

```diff
@@ -22,7 +22,7 @@ if(NOT TARGET OpenMP::OpenMP_CXX)
 endif()
 
 find_dependency(CUDA 9.0)
-find_dependency(OpenCV 3)
+find_dependency(OpenCV)
```

Comment:
Looks like this should have been changed [when I was adding compatibility with the Canonical-provided OpenCV version](https://github.com/ros-industrial/yak/commit/845a62a07aed23d3205f463592c67e013c8053c0), which I think is 2.4.